.gitignore

```diff
@@ -5,5 +5,4 @@ flagd
 dist/
 
 # ignore generated code
-pkg/generated/*.gen.*
```

Comment:
@James-Milligan thanks for this, this doesn't delete the file, but it at least makes it appear in my pending changes so I know to delete it!